GiniVision/Classes/CameraViewController.swift

```diff
@@ -71,11 +71,16 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     fileprivate var defaultImageView: UIImageView?
     // note that in the mapping left and right are reversed. That's because landscape values have different meanings
     // in UIDeviceOrientation and AVCaptureVideoOrientation. Refer to their documentaion for more info
-    fileprivate let orientationsMapping = [UIDeviceOrientation.portrait: AVCaptureVideoOrientation.portrait,
+    fileprivate let deviceOrientationsMapping = [UIDeviceOrientation.portrait: AVCaptureVideoOrientation.portrait,
                                            UIDeviceOrientation.landscapeRight: AVCaptureVideoOrientation.landscapeLeft,
                                            UIDeviceOrientation.landscapeLeft: AVCaptureVideoOrientation.landscapeRight,
                                            UIDeviceOrientation.portraitUpsideDown: AVCaptureVideoOrientation.portraitUpsideDown]
     
+    fileprivate let interfaceOrientationsMapping = [UIInterfaceOrientation.portrait: AVCaptureVideoOrientation.portrait,
```

Comment:
Why are BOTH of these two arrays necessary? 
Was it because there was a mismatch between landscape left and right on iPad?

Since this is becoming bigger and bigger, maybe it makes sense to extract it somewhere?